### DISCUSSION TITLE
How do you create an inscription like OnChainMonkey

### DISCUSSION THREAD

**Discussion by AronGM3** [ordinals/ord#1690] [Q&A] [answered]
I want to create a inscription like OCM did. Where you have multiple images into one inscription. I've been searching how to do it for hours but can't find it.

Link to the inscription:
https://ordinals.com/inscription/fb162a46943e5d7d31d72ee2c8c850e66c1ca5d0d453068aa63883528285ed21i0

Would love to make an inscription like theirs. Cheers

**Comment by niksrid** [comment, marked as answer]
Hint: 
Click on preview
Once it opens in the new tab
Right click and hit "View Page Source"
Et voila

**Reply by AronGM3** [reply]
Thanks for your help!

**Reply by huuep** [reply]
Hi @AronGM3, let me know if you need any help. The next step after the Collection Ordinal is https://github.com/casey/ord/discussions/1647 to distribute it efficiently.

**Reply by snotboogy** [reply]
!RISE

**Reply by swae09** [reply]
!RISE

**Reply by OCMJayB** [reply]
!RISE

**Reply by Fidenza** [reply]
Hint:
Click on preview
Once it opens in the new tab
Right click and hit "View Page Source"
Et voila

Where is the preview please revert answer. I am interested in the same answer

**Reply by BunnyNomics101** [reply]
Hey @huuep 
I know you're super busy with OCM but can I reach out for your input on a small project?
Be it git, Twitter or Discord. Thanks for your contribution!
